src/sentry/search/eap/columns.py

```diff
@@ -34,18 +35,26 @@ class ResolvedColumn:
     # Validator to check if the value in a query is correct
     validator: Callable[[Any], bool] | None = None
 
-    def process_column(row: Any) -> None:
-        """Pull the column from row, then process it and mutate it"""
-        raise NotImplementedError()
+    def process_column(self, value: Any) -> Any:
+        """Given the value from results, return a processed value if a processor is defined otherwise return it"""
+        if self.processor:
+            return self.processor(value)
+        return value
 
     def validate(self, value: Any) -> None:
         if self.validator is not None:
             if not self.validator(value):
                 raise InvalidSearchQuery(f"{value} is an invalid value for {self.public_alias}")
 
+    @property
+    def is_aggregate(self) -> bool:
+        """So that callers can easily identify if this resolved column is an aggregate or not"""
+        return isinstance(self.internal_name, Function.ValueType)
```

Comment:
Almost feels like we should be 2 classes `ResolvedColumn` and `ResolvedAggregate` instead of mashing the 2 into 1

Reply:
Hmm good point i'll consider this in a follow up 👍